### PR TITLE
Add blog coordinator handbook

### PR DESCRIPTION
**Pull request by fsmunoz**
Signed-off-by: Frederico Muñoz <fsmunoz@gmail.com>

**Which issue(s) this PR fixes**:

Add Blog Coordinator role handbook

### CODE REVIEW

**Comment by jberkus** [comment]
@kaslin please review.

/assign @kaslin

File: communication/marketing-team/role-handbooks/blog-coordinator.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+This handbook outlines expectations and responsibilities for Blog coordinator as well as Contributor Comms team members who wish to contribute to blogging activities.
+
+Writing and sharing articles is one of the ways through which Kubernetes contributors add to the community, and the Blog Coordinator role exists to
+
+1. Serve as an initial point of contact for content suggestions
+2. Help contributors to navigate the existing publishing processes
+3. Assist in determining the final publishing targets of the content.
+4. Coordinate with SIG-Docs in the publishing process, namely by following and improving the Blogging Guidelines.
+6. Coordinate with the rest of the Contributor Comms team, and specifically the Social Media Coordinator, in the best ways to broaden external awareness of published content.
```
Comment (reylejano) [inline communication/marketing-team/role-handbooks/blog-coordinator.md:11]:
```suggestion
5. Coordinate with the rest of the Contributor Comms team, and specifically the Social Media Coordinator, in the best ways to broaden external awareness of published content.
```

**Comment by reylejano** [comment]
Since PR https://github.com/kubernetes/community/pull/6911 has merged
Please fetch from upstream/master and rebase with @fsmunoz 's fsmunoz:blog-role branch 
Otherwise this PR will create the marketing-team subdir
Putting a hold until it's done so the marketing team subdir is not created
/hold

**Comment by fsmunoz** [comment]
@reylejano is this OK?

**Comment by reylejano** [comment]
looks good
/unhold
/lgtm

**Pull request merged**